cmd/generate.go

```diff
@@ -105,19 +105,20 @@ If multiple configurations are specified, each will be applied in left to right
 For example, 'fab generate prod east' will generate to a directory named prod-east.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		PrintVersion()
-
+		
```

Comment:
One nitpick - can you remove these added spaces from your PR.  A typical coding convention is not have trailing whitespace on lines because another person's editor might automatically remove these.  For example, I have the following set up in VS Code:  https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut

So the next time I edit this file, it will delete these spaces.  I'd recommend utilizing something like VSCode to automatically handle this for you as well.